openapi.yaml

```diff
@@ -1,6 +1,6 @@
 openapi: 3.0.1
 info:
-  version: 4.12.0
+  version: 4.13.0
```

Comment:
please double check the version to be sure. it was changed yesterday after I left.